Fiserv-IPG-Integracion-Consolidado.md

```
@@ -0,0 +1,4344 @@
+Paso 1 - Generar una cadena de texto con la siguiente estructura msgSignatureString = API_KEY + CLIENT_REQUEST_ID + TIMESTAMP + PAYLOAD.
+
+En donde:
+
+- API_KEY: Es la clave de acceso a la API provista por fiserv. Ejemplo: tOqWgOZAFq6aAYpqyQtAGVkjfo2Qp3lUxd
+- CLIENT_REQUEST_ID: Es un identificador único para cada transacción. Se recomienda utilizar el formato UUIDv4 universally unique identifier (UUID) de 128 bits hay diferentes librerías dependiendo el lenguaje de programación que utilicemos que nos podrían ayudar con esta tarea.
+- TIMESTAMP: Valor correspondiente al número de milisegundos transcurridos desde el 01/01/1970 hasta el instante en que se está realizando la petición. También se le conoce como Epoch time. Ejemplo: 1650590066714
+- PAYLOAD: Cuerpo/Body de la solicitud, el cual irá cambiando según el tipo de solicitud. En caso de que el Payload sea nulo, se debe considerar como una cadena vacía. El payload deberá ser serializado sin espacios en blanco ni saltos de línea como se muestra a continuación.
+
+| Payload Original | Payload enviado |
+| --- | --- |
+| `{`<br>`  “name”:”Fiserv”,`<br>`  “location”: “Jaime Balmes 11D”`<br>`}` | `{“name”:”Fiserv”,“location”:“Jaime Balmes 11D”}` |
```

markdownlint-cli2 (0.23.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)